Dockerfile.internal-platform

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.20-alpine3.17 AS builder
+
+ENV GO111MODULE=off CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO_PROJECT_PATH=github.com/bitrise-io/bitrise-webhooks
+
+COPY . $GOPATH/src/$GO_PROJECT_PATH
+
+RUN go install $GO_PROJECT_PATH
```

Comment:
this seems a bit weird, why move away from the usual `go mod download, go mod verify, go build -o something` path?

Reply:
it's an old project using dep and all the stuff before go modules. that's why I have to turn off go modules (https://github.com/bitrise-io/bitrise-webhooks/pull/127/files#diff-8f743f72d012b87f8f461727ec43e420aeb39261a67397e5f57a5b605d848dd3R3) and go with this weird go install method